sync/downstream.py

```diff
@@ -597,6 +607,8 @@ def try_push_complete(git_gecko, git_wpt, try_push, sync):
         disabled = []
         if not try_push.success():
             if sync.affected_tests:
+                if try_push.failure_limit_exceeded(sync):
```

Comment:
This seems wrong; it's perfectly fine for a non-stability push to fail all jobs.

Reply:
Instead I added a check for exceptions only to `try_push.wpt_tasks`. Less wrong?